src/Concordium/Client/GRPC2.hs

```diff
@@ -2533,6 +2536,303 @@ instance FromProto Proto.PendingUpdate where
                     PUETimeParameters <$> fromProto tParameters
         return PendingUpdate{..}
 
+type LoggerMethod = Text -> IO ()
```

Comment:
Perhaps this module should be split into two, a conversion-related and a client-logic related one. How do you feel about this?